dspace-server-webapp/src/test/java/org/dspace/app/rest/ResourcePolicyRestRepositoryIT.java

```diff
@@ -503,7 +504,9 @@ public void findResourcePoliciesOfOneResourceWithoutActionTest() throws Exceptio
             .andExpect(jsonPath("$.page.totalElements", is(2)));
     }
 
+    // This test is currently not working as intended, needs to be reviewed.
     @Test
+    @Ignore
```

Comment:
If this is going to be left in this PR, then we should create a JIRA ticket to describe the problem with the test, so that it can be assigned for a followup PR.